Detectors/TRD/simulation/src/Trap2CRU.cxx

```diff
@@ -659,7 +659,7 @@ void Trap2CRU::convertTrapData(o2::trd::TriggerRecord const& triggerrecord, cons
   //set startdigit and starttracklet relative to the trigger.
   //
   int starttrackletindex = triggerrecord.getFirstTracklet();
-  int endtrackletindex = triggerrecord.getFirstTracklet() + triggerrecord.getNumberOfTracklets();
+  int endtrackletindex = triggerrecord.getFirstTracklet() + triggerrecord.getNumberOfTracklets() - 1;
```

Comment:
Don't you skip the last tracklet in this way, taking into account the `<` in the https://github.com/AliceO2Group/AliceO2/blob/f7b4ceb7e4ff96158d9ee41420d4b4b17125a2b4/Detectors/TRD/simulation/src/Trap2CRU.cxx#L726 ?

Reply:
yes, it just seemed more elegant to put the minus 1 at the top. Would you prefer it there?

Reply:
I meant with this definition of `endtrackletindex` you are checking all but the last tracklet, why the last one should not be checked?

Reply:
The problem is actually at line 494 and its associated loop. The -1 solved my initial tests. So was hoping it was the actual problem.

Reply:
@shahor02 ok that fixes it properly, it fixes the valgrind memcheck, there is no "invalid read" but I cant test mac of course.

Reply:
it still breaks on mac.

Reply:
@TimoWilken the CI runs the full system test on mac as well or only centos8 ?